app/src/utils/streamSnippets.js

```diff
@@ -0,0 +1,32 @@
+export default (id) => ({
```

Comment:
The parameter could be an object here, i think this becomes more readable:
```
streamSnippets({
     id: dialogTargetStream.id,
})

Reply:
Maybe it does. I'll update. 👍